src/components/router/routerContext.js

```diff
@@ -121,26 +143,26 @@ const useRouteDetail = ({
   const product = useAliasSetRouteProduct();
   const { getBundleData = helpers.noop, updateDocumentTitle = helpers.noop } = useAliasChrome();
   const bundleData = getBundleData();
-  const productGroup = product?.productGroup;
```

Comment:
optional: inconsistent assignment use when compared to the updates for `toolbarFieldGroupVariant`